source/fuzz/fuzzer_util.h

```diff
@@ -493,6 +493,16 @@ opt::Instruction* GetLastInsertBeforeInstruction(opt::IRContext* ir_context,
                                                  uint32_t block_id,
                                                  SpvOp opcode);
 
+// Returns the id of an available local variable that has the type |type_id| and
+// is declared in the function that has the id |function_id|. If no such
+// variable is available, returns 0.
+uint32_t MaybeGetLocalVariable(opt::IRContext* ir_context, uint32_t type_id,
+                               uint32_t function_id);
```

Comment:
Mention that `type_id` is required to be a result id of `OpTypePointer` instruction with `Function` storage class.